swim/node_bootstrap_test.go

```diff
@@ -74,21 +74,20 @@ func (s *BootstrapTestSuite) TestBootstrapJoinsTimeOut() {
 }
 
 func (s *BootstrapTestSuite) TestBootstrapDestroy() {
-	var lerr lError
+	errChan := make(chan error)
```

Comment:
NOTES TO REVIEWER: Don't mind these changes. I cherry-picked a commit from https://github.com/uber/ringpop-go/pull/67 so that these tests pass. I'll delete this commit from this PR when that PR is merged.